.github/workflows/publish.yaml

```diff
@@ -12,6 +12,8 @@ jobs:
   publish:
     if: ${{ github.repository_owner == 'dart-lang' }}
     uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main
+    with:
+      ignore-packages: pkgs/cupertino_http,pkgs/cronet_http,pkgs/ok_http
     permissions:
       id-token: write # Required for authentication using OIDC
```

Comment:
@devoncarew - Are these config lines stale? I don't see the same in the other repos I checked.

Reply:
Latest from here: https://github.com/dart-lang/ecosystem/blob/main/pkgs/blast_repo/lib/src/tweaks/auto_publish_tweak.dart